strawberryfields/backends/tfbackend/backend.py

```diff
@@ -45,6 +44,10 @@ def __init__(self, graph=None):
             self._graph = tf.get_default_graph()
         else:
             self._graph = graph
+        # TODO docstrings
```

Comment:
Is this not part of current PR?

Reply:
Can be, but I'm not 100% familiar with the TF backend so if I write the docstrings they may be not perfectly accurate. Do you want to add them?

Reply:
why don't you have a go at writing them, then I can edit and update as needed?

Reply:
Sure.